node/cli/src/service.rs

```diff
@@ -115,22 +115,24 @@ impl Default for NetworkConfig {
     }
 }
 
-pub fn start_service<S>(
+pub struct DevNetConfig {
+    /// how often devnet produces blocks, in milliseconds
```

Comment:
it's not really milliseconds here.. it's just a duration

Reply:
Fixed in bb1ba6b